### PR TITLE
remove mention of seAp

### PR DESCRIPTION
**Pull request by rosejbon**
## Description

- Update site to remove any mention of seAp and replace with their new name The Advocacy People
-

@neontribe/ca-seap

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -82,9 +82,9 @@ brandCopy key =
                 { action = DesktopInteractive
                 , category = "email"
                 , icon = "email"
-                , href = "mailto:info@seap.org.uk"
-                , displayHref = "info@seap.org.uk"
-                , promptLong = "email info@seap.org.uk"
+                , href = "mailto: info@theadvocacypeople.org.uk"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: remove the space.

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -82,9 +82,9 @@ brandCopy key =
...
+                , displayHref = " info@theadvocacypeople.org.uk"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Do we want/need this space?

File: src/Copy/BrandCopy.elm [hunk 3]
```diff
@@ -82,9 +82,9 @@ brandCopy key =
...
+                , promptLong = "email  info@theadvocacypeople.org.uk"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: There is extra space

**Review by katjam — COMMENTED** [review]
Just a few stray spaces

File: src/Copy/BrandCopy.elm (hunk 2 above)
Comment (rosejbon) [reply src/Copy/BrandCopy.elm]: opps, no, will remove

**Review by katjam — APPROVED** [review]
Ace! :star2:

**Review by AlinChindea — APPROVED** [review]
:shipit:

**Pull request merged**